examples/stargate-cass311.yml

```diff
@@ -41,14 +41,18 @@ services:
       - CASSANDRA_SEEDS=backend-1
       - CASSANDRA_CLUSTER_NAME=backend
   stargate:
-    image: stargateio/stargate-3_11:v0.0.5
+    image: stargateio/stargate-3_11:v1.0.0
     container_name: stargate
     depends_on: 
       - backend-1
     networks:
       - backend
     ports:
       - 9045:9042
+      - 8080:8080
+      - 8081:8081
+      - 8082:8082
+      - 8085:8085
```

Comment:
Should be 8084 I believe

Reply:
Hmm... copied it from `examples/stargate-prometheus-grafana/docker-compose.yml`... should it be fixed there as well?

Reply:
Probably a good idea. I forgot to come over and check these after @olim7t 's admin port changes.

Reply:
ok, will do this kind of change across all examples here